### PR TITLE
Ch11: 11.5 Minibatch-sgd & 11.6 Momentum

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-948/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-948/

**Comment by goldmermaid** [comment]
Forum links checked;
cite, numref checked.

File: chapter_optimization/minibatch-sgd.md [hunk 1]
```diff
@@ -1,30 +1,57 @@
...
 
-首先，具有 16 个内核和 AVX-512 矢量化的 2GHz CPU 每秒可处理高达 $2 \cdot 10^9 \cdot 16 \cdot 32 = 10^{12}$ 个字节。GPU 的能力很容易超过该数字 100 倍。另一方面，中端服务器处理器的带宽可能不超过 100 Gb/s，即不到保持处理器供给所需的十分之一。更糟糕的是，并非所有的内存访问都是平等的：内存接口通常为 64 位或更宽（例如，在最多 384 位的 GPU 上），因此读取单个字节会产生更广泛访问的成本。
+首先，具有 16 个内核和 AVX-512 矢量化的 2GHz CPU 每秒可处理高达 $2 \cdot 10^9 \cdot 16 \cdot 32 = 10^{12}$ 个字节。
+同时，GPU 的性能很容易超过该数字 100 倍。
+而另一方面，中端服务器处理器的带宽可能不超过 100 Gb/s，即不到处理器满负荷所需的十分之一。
+更糟糕的是，并非所有的内存入口都是相等的：内存接口通常为 64 位或更宽（例如，在最多 384 位的 GPU 上）。
+因此读取单个字节会导致由于更宽的存取而产生的成本。
 
-其次，第一次访问的开销很大，而顺序访问相对便宜（这通常称为突发读取）。还有很多事情要记住，比如当我们有多个套接字、切片和其他结构时进行缓存。有关更深入的讨论，请参阅此 [Wikipedia article](https://en.wikipedia.org/wiki/Cache_hierarchy)。
+其次，第一次存取的额外开销很大，而按序存取（sequential access）或突发读取（burst read）相对便宜。
```
Comment (astonzhang) [inline chapter_optimization/minibatch-sgd.md:30]: 便宜 -> 开销较小

File: chapter_optimization/minibatch-sgd.md [hunk 2]
```diff
@@ -1,30 +1,57 @@
...
+有关更深入的讨论，请参阅此 [维基百科文章](https://en.wikipedia.org/wiki/Cache_hierarchy)。
 
-缓解这些限制的方法是使用实际上足够快的 CPU 缓存层次结构，以便为处理器提供数据。这是深度学习中批量处理背后的 * 推动力。为了简单起见，请考虑矩阵-矩阵乘法，比如 $\mathbf{A} = \mathbf{B}\mathbf{C}$。我们有很多选择来计算 $\mathbf{A}$。例如，我们可以尝试以下方法：
+减轻这些限制的方法是使用足够快的 CPU 缓存层次结构来为处理器提供数据。
+这是深度学习中批量处理背后的推动力。
+举一个简单的例子：矩阵-矩阵乘法。
+比如 $\mathbf{A} = \mathbf{B}\mathbf{C}$，我们有很多方法来计算 $\mathbf{A}$。例如，我们可以尝试以下方法：
 
-1. 我们可以计算 $\mathbf{A}_{ij} = \mathbf{B}_{i,:} \mathbf{C}_{:,j}^\top$，也就是说，我们可以通过点积进行元素计算。
+1. 我们可以计算 $\mathbf{A}_{ij} = \mathbf{B}_{i,:} \mathbf{C}_{:,j}^\top$，也就是说，我们可以通过点积进行逐个元素计算。
```
Comment (astonzhang) [inline chapter_optimization/minibatch-sgd.md:38]: 逐元素计算

File: chapter_optimization/minibatch-sgd.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_optimization/minibatch-sgd.md:30]: resolved at https://github.com/d2l-ai/d2l-zh/pull/952

File: chapter_optimization/minibatch-sgd.md (hunk 2 above)
Comment (goldmermaid) [reply chapter_optimization/minibatch-sgd.md:38]: resolved at https://github.com/d2l-ai/d2l-zh/pull/952

**Pull request merged**